### PR TITLE
Fixes #26188 - don't duplicate -s in the python shebang

### PR DESCRIPTION
**Pull request by evgeni**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

This requires https://github.com/Katello/katello-certs-tools/pull/9 to build properly

### CODE REVIEW

File: packages/foreman/katello-certs-tools/katello-certs-tools.spec [hunk 1]
```diff
@@ -42,10 +36,8 @@ pathfix.py -pni "%{__python3} %{py3_shbang_opts}" . katello-ssl-tool
 %install
 %if 0%{?rhel} == 7
 %{__python2} setup.py install --skip-build --root $RPM_BUILD_ROOT
```
Comment (ekohl) [inline packages/foreman/katello-certs-tools/katello-certs-tools.spec]: Can this now use `%{py2_install}`?

Comment (evgeni) [reply packages/foreman/katello-certs-tools/katello-certs-tools.spec]: Why couldn't it before? I have no idea :)

File: packages/foreman/katello-certs-tools/katello-certs-tools.spec [hunk 2]
```diff
@@ -12,7 +12,6 @@ Requires: openssl
 Requires: rpm-build
 
 BuildRequires: docbook-utils
-BuildRequires: /usr/bin/pathfix.py
```
Comment (ehelms) [inline packages/foreman/katello-certs-tools/katello-certs-tools.spec]: Why are is this being dropped? There are still shebangs.

Comment (evgeni) [reply packages/foreman/katello-certs-tools/katello-certs-tools.spec]: Only in `katello-ssl-tool` itself, and that gets handled just fine by `%{pyX_build}` which adds `--executable=/usr/bin/python2 -s`. see http://koji.katello.org/kojifiles/work/tasks/6253/176253/build.log for an example

**Comment by evgeni** [comment]
:dancer:

**Comment by evgeni** [comment]
release build: http://koji.katello.org/koji/taskinfo?taskID=176286

**Pull request merged**